browser/ui/webui/brave_adblock_ui.cc

```diff
@@ -111,37 +113,7 @@ void AdblockDOMHandler::HandleUpdateCustomFilters(const base::ListValue* args) {
 BraveAdblockUI::BraveAdblockUI(content::WebUI* web_ui, const std::string& name)
     : BasicUI(web_ui, name, kBraveAdblockGenerated,
         kBraveAdblockGeneratedSize, IDR_BRAVE_ADBLOCK_HTML) {
-  Profile* profile = Profile::FromWebUI(web_ui);
-  PrefService* prefs = profile->GetPrefs();
-  pref_change_registrar_ = std::make_unique<PrefChangeRegistrar>();
```

Comment:
we don't need `#include "components/prefs/pref_change_registrar.h"`, pls check for other unused headers

Reply:
done.